libs/health-insurance/src/lib/bucket/bucket.service.ts

```diff
@@ -0,0 +1,65 @@
+import { Inject, Injectable } from '@nestjs/common'
+import * as AWS from 'aws-sdk'
+import * as S3 from 'aws-sdk/clients/s3'
+import { Logger, LOGGER_PROVIDER } from '@island.is/logging'
+
+// TODO: get bucket name from url - STOP HARD CODE
+const BUCKET_NAME = 'island-is-dev-storage-application-system'
+const REGION = 'eu-west-1'
+
+const s3 = new AWS.S3({
+  region: REGION,
```

Comment:
Region should be configured from the environment/profile.

Reply:
Same as above